### DISCUSSION TITLE
Multiplayer features not working?

### DISCUSSION THREAD

**Discussion by Theighlin** [ppy/osu#16983] [Q&A] [answered]
I'm really not sure about opening this discussion, but i don't know what better place to ask.
Is it intended for multiplayer not to be working at this moment?
The status page stopped reporting stats about 6 hours ago:
![image](https://user-images.githubusercontent.com/53081750/155578240-018b69f2-c25b-4fc2-bbbc-f143c2af1788.png)
![image](https://user-images.githubusercontent.com/53081750/155578346-d45bde41-f0f3-46dd-b613-c217493acdd9.png)
![image](https://user-images.githubusercontent.com/53081750/155578412-3c0e226e-b996-4455-9309-252ee9ea2190.png)

[network.log](https://github.com/ppy/osu/files/8134651/network.log)
[runtime.log](https://github.com/ppy/osu/files/8134652/runtime.log)

**Comment by bdach** [comment, marked as answer]
Thanks, I can corroborate this. Have escalated this internally for investigation.

**Reply by bdach** [reply]
Everything should be back up now.
